linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/DashboardHandler.scala

```diff
@@ -6,19 +6,22 @@ import com.twitter.util.Future
 import io.buoyant.linkerd.Build
 
 private[admin] class DashboardHandler extends Service[Request, Response] {
-  lazy val html = dashboardHtml
+  import DashboardHandler._
 
   override def apply(req: Request): Future[Response] = req.path match {
     case "/" =>
-      AdminHandler.mkResponse(html)
+      AdminHandler.mkResponse(dashboardHtml())
+    case rexp(router) =>
+      AdminHandler.mkResponse(dashboardHtml(router))
     case _ =>
       Future.value(Response(Status.NotFound))
   }
 
-  def dashboardHtml = {
+  def dashboardHtml(router: String = "") = {
     AdminHandler.html(
       content = s"""
         <div class="request-totals"></div>
+        <div class="router-name-data" data-router-name="${router}" style="visibility:hidden"></div>
```

Comment:
just curious: what's the difference between doing this vs `<script id="data-router-name" type="application/json">$router</script>`?